http/headers/feature-policy.json

```diff
@@ -1853,6 +1853,56 @@
             }
           }
         },
+        "web-share": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/HTTP/Headers/Feature-Policy/web-share",
+            "support": {
+              "chrome": {
+                "version_added": null
+              },
+              "chrome_android": {
+                "version_added": null
+              },
+              "edge": {
+                "version_added": null
+              },
+              "firefox": {
+                "version_added": "81",
+                "notes": "Firefox 81 enables support for the Feature-Policy web-share, but does not support the WebShare API. See <a href='https://bugzil.la/1653481'>bug 1653481</a>."
+              },
+              "firefox_android": {
+                "version_added": "81",
+                "notes": "Firefox 81 enables support for the Feature-Policy web-share, but does not support the WebShare API. See <a href='https://bugzil.la/1653481'>bug 1653481</a>."
+              },
+              "ie": {
+                "version_added": false
+              },
+              "opera": {
+                "version_added": null
+              },
+              "opera_android": {
+                "version_added": null
+              },
+              "safari": {
+                "version_added": null
```

Comment:
I couldn't find evidence that WebKit has implemented this: https://bugs.webkit.org/show_bug.cgi?id=214448. I'd set both Safaris to `false` too.